src/isomorphic/hooks/ReactComponentTreeHook.js

```diff
@@ -39,6 +43,24 @@ function isNative(fn) {
   }
 }
 
+type Item = {
+  element: ReactElement,
+  parentID: DebugID,
+  text: ?string,
+  childIDs: Array<DebugID>,
+  isMounted: bool,
+  updateCount: number,
+};
+
+declare function getItem(id: DebugID): ?Item;
```

Comment:
Looks like I don't even need this block anymore, if I remove it, it works with flow!